corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1633,7 +1633,9 @@ def _report_charts(self):
                 return [{
                     "type": "multibar",
                     "x_axis_column": agged_columns[0]['column_id'] if agged_columns else '',
-                    # TODO: Possibly use more columns?
+                    # Populate only the top level columns here.
```

Comment:
as opposed to what?

Reply:
top level are the ones that are defined in the report as the columns to be shown.
When we use "Count per choice" (aka expanded columns), it fetches all the choices from the table and adds each of those as a column.

https://github.com/dimagi/commcare-hq/blob/3e8612b30abb9b01455c6692bb824dc9836a5f1d/corehq/apps/userreports/mixins.py#L59-L65

https://github.com/dimagi/commcare-hq/blob/cd1984bd15a93a9bcfb9abaa0a276671534bce08/corehq/apps/userreports/sql/columns.py#L21-L41